src/backends/interflop-mca/Makefile.am

```diff
@@ -1,5 +1,5 @@
 lib_LTLIBRARIES = libinterflop_mca.la
 libinterflop_mca_la_SOURCES = interflop_mca.c
-libinterflop_mca_la_LDFLAGS = -lm
+libinterflop_mca_la_LDFLAGS = -lm -lquadmath
```

Comment:
You should remove -lquadmath and add a test for checking if `__float128` is supported by the compiler.
It removes dependency and gets the configure.ac cleaner.

Reply:
Sure. I removed the dependency and check in configure.ac for now.
Checking for __float128 is not trivial, I will open an issue and move it to another PR.

Reply:
Opened issue #110